stable/chaoskube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: chaoskube
-version: 0.14.0
-appVersion: 0.12.1
+version: 0.15.0
```

Comment:
With #11780 this version number should change to `1.1.0`

Reply:
Thanks, @marc-sensenich for the update.
@linki would you want me to update this to `1.1.0` ?